beacon_node/lighthouse_network/src/rpc/mod.rs

```diff
@@ -310,9 +310,13 @@ where
                         Err(RateLimitedErr::TooLarge) => {
                             // we set the batch sizes, so this is a coding/config err for most protocols
                             let protocol = req.versioned_protocol().protocol();
-                            if matches!(protocol, Protocol::BlocksByRange)
-                                || matches!(protocol, Protocol::BlobsByRange)
-                            {
+                            if matches!(
+                                protocol,
+                                Protocol::BlocksByRange
+                                    | Protocol::BlobsByRange
+                                    | Protocol::BlocksByRoot
+                                    | Protocol::BlobsByRoot
+                            ) {
```

Comment:
The log below need to be updated too - remove "By range", and may be worth mentioning that request is too large

Reply:
Fixed in [907dc34](https://github.com/sigp/lighthouse/pull/5381/commits/907dc3407be797afa9db5414411a305a7aca6868)